Cargo.lock

```diff
@@ -1,6 +1,6 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
-version = 3
+version = 4
```

Comment:
@Rickvanderveen, I don't think there's anything fundamentally wrong with this PR; at a quick glance, it would appear that the update to the `Cargo.lock` version might be interacting with some older CI versions. Do you want to try downgrading this to `3`? We'll have to move forward at some point but that doesn't need to happen here.